src/wallet/scriptpubkeyman.h

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) 2019 The Bitcoin Core developers
+// Copyright (c) 2020 The PIVX Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef PIVX_SCRIPTPUBKEYMAN_H
+#define PIVX_SCRIPTPUBKEYMAN_H
+
+#include "wallet/hdchain.h"
+#include "wallet/wallet.h"
+#include "wallet/walletdb.h"
+
+//! Default for -keypool
+static const unsigned int DEFAULT_KEYPOOL_SIZE = 1000;
```

Comment:
this can be probably lowered to 100 as already discussed

Reply:
done